extensions/python/tests/PythonManifestTests.cpp

```diff
@@ -157,7 +157,11 @@ class MyPyProc5(FlowFileTransform):
 )";
 
   controller.configuration_->set(minifi::Configuration::nifi_python_processor_dir, python_dir.string());
+#ifdef WIN32
   controller.configuration_->set(minifi::Configuration::nifi_extension_path, "*minifi-python-script*");
+#else
+  controller.configuration_->set(minifi::Configuration::nifi_extension_path, "*minifi-python-lib-loader*, *minifi-python-script*");
```

Comment:
this is only relevant on Linux, and probably shouldn't be included on macos

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1916/commits/ddcb6a3d516b5472a47af0e521553e95ad463b5e